src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -350,13 +356,30 @@ def makeTransferRequest(self, wflow):
                 self.logger.info(msg)
 
                 if self.msConfig.get('enableDataTransfer', True):
+                    # Force request-only subscription
+                    # to any data transfer going above some threshold (do not auto-approve)
+                    aboveWarningThreshold = self.msConfig.get('warningTransferThreshold') > 0. and \
+                        dataSize > self.msConfig.get('warningTransferThreshold')
+                    if aboveWarningThreshold and subscription.request_only != "y":
+                        subscription.request_only = "y"
+
                     # Then make the data subscription, for real!!!
                     success, transferId = self._subscribeData(subscription, wflow.getName(), dataIn['name'])
                     if not success:
                         break
                     if transferId:
                         transRec['transferIDs'].add(transferId)
 
+                    # Warn about data transfer subscriptions going above some treshold
+                    if aboveWarningThreshold:
+                        emailSubject = "Large data transfer in place for dataset: {dset}".format(dset=dataIn['name'])
```

Comment:
It might be useful to start the subject with `[MS] Large data ...`
And in the line below, I believe I've read somewhere (maybe from pylint report) that triple simple quotes should be avoided, instead use triple double quotes please.